packages/babel-plugin-transform-class-properties/test/fixtures/general/static-infer-name/expected.js

```diff
@@ -0,0 +1,5 @@
+var _class, _temp;
+
+var Foo = (_temp = _class = function Foo() {
+  babelHelpers.classCallCheck(this, Foo);
```

Comment:
Seems the helper is also replacing `_class` here with `Foo`, so I think this should be `babelHelpers.classCallCheck(this, _class);`. Although I'm not sure if this has any implications at all.